### PR TITLE
fix(libs): Revise file name/structure for client and server components

### PR DESCRIPTION
**Pull request by xlisachan**
## This pull request

- updates Gruntfile as well as  file names (to PascalCase) and file structure as seen below:
   - `<component>`
      - `en.ftl`
      - `index.tsx` 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.

### CODE REVIEW

File: apps/payments/next/Gruntfile.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ module.exports = function (grunt) {
   const srcPaths = [
     '.license.header',
     'app/**/*.ftl',
```
Comment (StaberindeZA) [inline apps/payments/next/Gruntfile.js:8]: Should this be updated to also only include `en.ftl` files?

**Review by StaberindeZA — APPROVED** [review]
r+. lgtm

**Review by bcolsson — APPROVED** [review]
Approving. It looks like this isn't really changing anything for localization / exposing strings, but please let me know if I'm mistaken.

**Comment by xlisachan** [comment]
Yep, this PR is just cleaning up the filenames and file structure, no changes to strings have been made. Thanks @bcolsson! :)

**Pull request merged**